### PR TITLE
Added snapshot table clean up to CommonEventuateJdbcAccessImplTest. S…

### PR DESCRIPTION
**Pull request by dartartem**
…ee: https://github.com/eventuate-local/eventuate-local-quarkus/issues/5

### CODE REVIEW

File: eventuate-client-java-jdbc-common-tests/src/main/java/io/eventuate/javaclient/jdbc/common/tests/CommonEventuateJdbcAccessImplTest.java [hunk 1]
```diff
@@ -33,6 +33,7 @@
   protected void clear() {
     getEventuateJdbcStatementExecutor().update(String.format("delete from %s", getEventuateSchema().qualifyTable("events")));
     getEventuateJdbcStatementExecutor().update(String.format("delete from %s", getEventuateSchema().qualifyTable("entities")));
+    getEventuateJdbcStatementExecutor().update(String.format("delete from %s", getEventuateSchema().qualifyTable("snapshots")));
```
Comment (cer) [inline eventuate-client-java-jdbc-common-tests/src/main/java/io/eventuate/javaclient/jdbc/common/tests/CommonEventuateJdbcAccessImplTest.java:36]: This is such obvious copy/paste - please fix: `truncateTable("events")`

File: eventuate-client-java-micronaut-jdbc/build.gradle [hunk 2]
```diff
@@ -40,3 +40,7 @@ dependencies {
 test {
     useJUnitPlatform()
 }
+
+test {
```
Comment (cer) [inline eventuate-client-java-micronaut-jdbc/build.gradle:44]: Why?

File: eventuate-client-java-spring-jdbc/build.gradle [hunk 3]
```diff
@@ -14,3 +14,7 @@ dependencies {
     testCompile project(":eventuate-client-java-test-util")
     testCompile "org.springframework.boot:spring-boot-starter-test:$springBootVersion"
 }
+
+test {
+    forkEvery 1
```
Comment (cer) [inline eventuate-client-java-spring-jdbc/build.gradle:19]: Why>

**Review by cer — CHANGES_REQUESTED** [review]

File: eventuate-client-java-jdbc-common-tests/src/main/java/io/eventuate/javaclient/jdbc/common/tests/CommonEventuateJdbcAccessImplTest.java (hunk 1 above)
Comment (dartartem) [reply eventuate-client-java-jdbc-common-tests/src/main/java/io/eventuate/javaclient/jdbc/common/tests/CommonEventuateJdbcAccessImplTest.java:36]: Sorry, I will fix

**Pull request merged**